NDTensors/src/lib/SparseArrayDOKs/test/runtests.jl

```diff
@@ -94,5 +95,34 @@ using SparseArrays: SparseMatrixCSC, nnz
       end
     end
   end
+
+  @testset "Maybe Grow Feature" begin
+    a = SparseArrayDOK{elt,2}((0, 0))
+
+    setindex_maybe_grow!(a, 230, 2, 3)
```

Comment:
```suggestion
    SparseArrayDOK.setindex_maybe_grow!(a, 230, 2, 3)
```
and remove it from the list above, to make it clearer it is meant to be a private function.